aten/src/ATen/BatchedFallback.cpp

```diff
@@ -55,9 +55,9 @@ void batchedTensorForLoopFallback(const c10::OperatorHandle& op, torch::jit::Sta
   TORCH_CHECK(areAllReturnsTensors(schema) && !areAnyArgumentsTensorList(schema),
               "Batching rule not implemented for ", schema, ". ",
               "We could not generate a fallback.");
-  TORCH_CHECK(num_returns == 1,
+  TORCH_CHECK(num_returns >= 1,
               "Batching rule not implemented for ", schema, ". ",
-              "We do not yet support operations with multiple returns.");
+              "We do not support operations with no returns.");
```

Comment:
shouldn't this be easy to support lol

Reply:
Yeah, this isn't hard to support. However it implies the operation does nothing (since it would have to have no returns and no alias annotations). I'll clarify that this means that the fallback doesn't support that